.adjustVersion/.releaserc

```diff
@@ -0,0 +1,18 @@
+{
+  "branches": ["release/*"],
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    [
+      "@semantic-release/exec",
```

Comment:
This would be cleaner using `@google/semantic-release-replace-plugin`, it's well reasoned and plays well with all the other plugins

- [Example Usage](https://github.com/algorandfoundation/TEALScript/blob/d7d4e8c4bebd4de1a384be73239e0de2e799eb0e/package.json#L100C10-L100C49)